src/intl/locales/en.json

```diff
@@ -310,5 +310,41 @@
   {
     "id": "form.indiehackers",
     "defaultMessage": "Your IndieHackers profile"
+  },
```

Comment:
@JKorum just making sure that you know about the `@formatjs/cli` package.

You can run the following (from the root of the repository) to *automatically* extract these message definitions from your code:
```
$ npm i -g @formatjs/cli
$ formatjs extract --out-file src/intl/locales/en.json ./src/*/*.ts*
```